docs/events/timeline-ui-overview.asciidoc

```diff
@@ -162,3 +162,18 @@ the Timeline `ndjson` file.
 +
 NOTE: Each Timeline object in the file must be represented in a single line.
 Multiple Timeline objects are delimited with newlines.
+
+[discrete]
+[[filter-with-eql]]
+== Filter Timeline Results with EQL
+Use the *Correlations* tab to investigate Timeline results with EQL queries. You can enter EQL queries into the free-form text box under this tab and a link to EQL documentation is available in the bottom right corner of the text box.
+
+When forming EQL queries, you can write a basic query to return a list of events and alerts. Or, you can create sequences of EQL queries to view matched, ordered events across multiple event categories. Sequence queries are useful for identifying and predicting related events. They can also provide a more complete picture of potential adversary behavior in your environment, which you can use to create or update rules and detection alerts.
+
+The following image shows what matched ordered events look like in the Timeline table. Events are paired into sets of two and shaded red or blue to indicate that they are matched:
+
+From the *Correlations* tab you can also perform the following:
+
+* Specify the span of time that you want to investigate.
+* Re-order the columns and choose which fields display.
+* Choose whether you want to see events, detection alerts, all data sources, or a custom data source. All data sources are displayed by default.
```

Comment:
Consider adding the following statement / hyperlink:

> Custom data sources may include [Kibana index patterns](https://www.elastic.co/guide/en/kibana/current/index-patterns.html).

and re-ordering the enumerated list of data sources so the default, `All data sources`, is listed first, and the others follow in the same order that they appear in the UI:

`Choose whether you want to see All data sources, just events, just detection alerts, or a custom data source. Custom data sources may include Kibana index patterns. All data sources are displayed by default.`